backend/lib/services/members.js

```diff
@@ -44,9 +44,9 @@ function fromResource (project = {}, serviceAccounts = []) {
     .chain(project)
     .get('spec.members')
     .filter(['kind', 'User'])
-    .map('name')
-    .map(username => ({
+    .map(({ name: username, role, roles }) => ({
```

Comment:
Don't understand that role, roles thing. Is this done beacuse of backward compatibility?

Reply:
yes